connector/src/main/resources/config.yml

```diff
@@ -103,4 +103,4 @@ metrics:
   uuid: generateduuid
 
 # DO NOT TOUCH!
-config-version: 3
+config-version: 4
```

Comment:
This didn't need to be changed as no new config options where added just a new value and updated comment